test/Microsoft.Framework.Internal.Test/BufferEntryCollectionTest.cs

```diff
@@ -0,0 +1,167 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace Microsoft.Framework.Internal
+{
+    public class BufferEntryCollectionTest
+    {
+        [Fact]
+        public void Add_AddsBufferEntries()
+        {
+            // Arrange
+            var collection = new BufferEntryCollection();
+            var inner = new BufferEntryCollection();
```

Comment:
add something to this collection to demonstrate the final `collection.Add()` call isn't a no-op